pkg/systemstatsmonitor/system_stats_monitor.go

```diff
@@ -69,6 +70,8 @@ func NewSystemStatsMonitorOrDie(configPath string) types.Monitor {
 		glog.Fatalf("Failed to apply configuration for %q: %v", configPath, err)
 	}
 
+	glog.Infof("Error: %v", ssm.config)
```

Comment:
`glog.Errorf`?

Reply:
I added for debugging. Removed it as it is not required.

Reply:
Did you mean to remove this?